benchmark/brotli/creation.js

```diff
@@ -0,0 +1,27 @@
+// Flags: --expose-brotli
+'use strict';
+const common = require('../common.js');
+const brotli = require('brotli');
```

Comment:
I'd generally much prefer a much more generic module namespace for this... e.g. `require {brotli} = require('compression')` or some such, but I can live with this.

Reply:
I think `@node/compression` (or similar) would be good, because the `compression` package is quite popular.

Reply:
I like the idea of the scope. we can alias other compression modules into there as well